core/osm_connector.py

```diff
@@ -259,48 +270,63 @@ def get_stops(self, refresh=False):
 
         return self.stops
 
-    def _build_route_master(self, route_master, members):
-        """Helper function to build a RouteMaster object
+    def _build_line(self, route_master, itineraries):
+        """Helper function to build a Line object
 
-        Returns a initiated RouteMaster object from raw data
+        Returns a initiated Line object from raw data
 
         """
         if 'ref' in route_master.tags:
             ref = route_master.tags['ref']
         else:
             sys.stderr.write(
-                "RouteMaster without 'ref'. Please fix in OpenStreetMap\n")
+                "Relation without 'ref'. Please fix in OpenStreetMap\n")
             sys.stderr.write(
                 "http://osm.org/relation/" + str(route_master.id) + "\n")
 
-            # Check if a ref can be taken from one of the members
+            # Check if a ref can be taken from one of the itineraries
             ref = False
-            for member in list(members.values()):
-                if not ref and member.ref:
-                    ref = member.ref
+            for itinerary in list(itineraries.values()):
+                if not ref and itinerary.ref:
+                    ref = itinerary.ref
                     sys.stderr.write(
                         "Using 'ref' from member variant instead\n")
-                    sys.stderr.write(
-                        "http://osm.org/relation/" + str(member.id) + "\n")
+                    sys.stderr.write(itinerary.osm_url + "\n")
 
             # Ignore whole Line if no reference number could be obtained
             if not ref:
                 sys.stderr.write(
-                    "No 'ref' could be obtained from members. Skipping.\n")
+                    "No 'ref' could be obtained. Skipping whole route.\n")
                 return
 
         name = route_master.tags['name']
         frequency = None
         if "frequency" in route_master.tags:
             frequency = route_master.tags['frequency']
-        rm = RouteMaster(route_master.id, ref, name, members, frequency)
-        print(rm)
-        return rm
 
-    def _build_route_variant(self, route_variant, query_result_set, rm=None):
-        """Helper function to build a RouteVariant object
+        if 'route_master' in route_master.tags:
+            route_type = route_master.tags['route_master'].capitalize()
+
+        # If there was no route_master present we have a route relation here
+        elif 'route' in route_master.tags:
+            route_type = route_master.tags['route'].capitalize()
+
+        print route_type
```

Comment:
stray print? Btw. please let's aim for python 3 already so the transition will be easy.